src/current/_includes/releases/v24.1/v24.1.0-rc.1.md

```diff
@@ -0,0 +1,57 @@
+
+<h3 id="v24-1-0-rc-1-db-console-changes">DB Console changes</h3>
+
+- Added a **Replication Lag** graph, which measures replication lag between primary and standby clusters for [physical cluster replication]({% link v24.1/physical-cluster-replication-overview.md %}), to the [**Physical Cluster Replication**]({% link v24.1/physical-cluster-replication-monitoring.md %}) dashboard. [#123285][#123285]
+
+<h3 id="v24-1-0-rc-1-bug-fixes">Bug fixes</h3>
+
+- The [**Tables**]({% link v24.1/ui-databases-page.md %}#tables-view) and [**Table details**]({% link v24.1/ui-databases-page.md %}#table-details) pages in the DB Console now display the correct **Table Stats Last Updated** value. [#122816][#122816]
+- Fixed a bug in the DB Console's [**Custom Chart**]({% link v24.1/ui-custom-chart-debug-page.md %}) tool where store-level metrics were not being displayed properly. Previously, if a store-level metric was selected to be displayed at the store-level on a multi-store node, only data for the 1st store ID associated with that node would be displayed.  This patch ensures that data is displayed for all stores present on a node. Additionally, it updates the behavior to show a single time series for each store, as opposed to aggregating (e.g., `SUM`) all stores across the node. This allows finer-grained observability into store-level metrics. [#122705][#122705]
+- Previously, CockroachDB could run into `attempting to append refresh spans after the tracked timestamp has moved forward` internal errors in some edge cases. This is now fixed. The bug had been present since v22.2. [#123136][#123136]
+- Fixed a bug where `TYPEDESC SCHEMA CHANGE` jobs could end up retrying forever if the descriptor targeted by them was already dropped. [#123273][#123273]
+- Previously, if the legacy schema changer was being used, the [`CREATE SEQUENCE`]({% link v24.1/create-sequence.md %}) command would incorrectly check if the user had the `CREATE` privilege on the parent database. This check is now removed, and the `CREATE SEQUENCE` command only needs `CREATE` privilege on the parent schema now. [#123289][#123289]
+- A job will now log rather than fail if it reports an out-of bound progress fraction. [#122965][#122965]
+
+<h3 id="v24-1-0-rc-1-performance-improvements">Performance improvements</h3>
+
+- Added a new session setting `optimizer_use_improved_zigzag_join_costing`. When enabled, the cost of zigzag joins is updated so they will be never be chosen over scans unless they produce fewer rows. This change only matters if the setting `enable_zigzag_join` is also true. [#123106][#123106]
```

Comment:
```suggestion
- Added a new session setting `optimizer_use_improved_zigzag_join_costing`. When enabled and when the cluster setting `enable_zigzag_join` is also enabled, the cost of zigzag joins is updated, and a zigzag join will be used only if it produces fewer rows than a scan.[#123106][#123106]
```
Link to "session setting" (and to "cluster setting" if you also take the suggestion)?